inst/tests/tests.Rraw

```diff
@@ -8913,7 +8913,7 @@ test(1622.1, ans1, ans2)
 test(1622.2, ans1, fread(testDir("issue_1573_fill.txt"), fill=TRUE, sep=" ", na.strings=""))
 
 # fix for #989
```

Comment:
```suggestion
# fix for #989; possibly skip for #6576
```

Reply:
(GitHub can't apply suggested changes to this file from the UI, I guess because it's so huge, so you'll have to apply it in a new commit, sorry about that)